python/ray/tune/integration/torch.py

```diff
@@ -127,13 +162,17 @@ def load_checkpoint(self, checkpoint_dir: str):
 
     def stop(self):
         ray.get([worker.stop.remote() for worker in self.workers])
+        if self.should_colocate:
+            remove_placement_group(self._placement_group)
 
 
 def DistributedTrainableCreator(
         func: Callable,
         use_gpu: bool = False,
         num_workers: int = 1,
-        num_cpus_per_worker: int = 1,
+        cpus_per_worker: int = 1,
+        gpus_per_worker: Optional[int] = None,
+        workers_per_node: Optional[int] = None,
```

Comment:
I'm still not sure if workers_per_node is necessary. Couldn't each bundle just represent the resources for 1 worker (instead of all the workers on a node), and then use SPREAD strategy instead of STRICT_SPREAD?

Reply:
discussed offline